.github/workflows/deploy.yml

```diff
@@ -0,0 +1,122 @@
+name: Deploy
+
+on:
+  push:
+    branches: [ beta, main ]
+
+jobs:
+  publish:
+    name: Publish
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Read node version (package.json)
+        run: echo "::set-output name=version::$(jq -r .engines.node package.json)"
+        id: node-version
+
+      - name: Use Node.js (package.json)
+        uses: actions/setup-node@v1
+        with:
+          node-version: "${{ steps.node-version.outputs.version }}"
```

Comment:
On fuelrats.com, we get our node version from .nvmrc. Since that file doesn't exist here, I decided to use your engine version instead. This way GHA will always use the correct node version without you having to touch the action files.

Side note, I originally tried using an action to extract it for me, but that didn't work so jq was used to parse and extract the value instead. I've never used this package before. it's really cool.